services/shell/client_test.go

```diff
@@ -0,0 +1,59 @@
+package shell_test
```

Comment:
The Shell service had no pre-existing `client_test`. I added one, but it only tests `DoCommand` functionality.